### ISSUE TITLE
memory issues with iis worker process

### ISSUE THREAD

**Issue by ravilog** [StefH/KendoGridBinderEx#46]
Hi,  Thank you for providing this wonderful tool under MIT license. 

We have been using this in our app, and lately we found that whenever we run the page that binds kendo grid, there is a huge spike in the w3wp.exe memory and it does not subside even after some hours until app recycle kicks in.

**Comment by StefH** [comment]
@ravilog 
What you could try is upgrading the System.Linq.Dynamic.Core library to the latest ?

**Comment by ravilog** [comment]
Thanks for the suggestion currently we are using version 1.0.8.18. And also we are passing method instance as source as an AsQueryable item to the KendoGrid instance. Does it susceptible to cause any memory leak?

Note: worker process memory does look optimal when the grid page is not accessed.

**Comment by maulik-modi** [comment]
Yes, @ravilog , you have nailed down the issue. Try to convert method instance as static helper perhaps. please post some code to guide correctly.
